features/0036-issue-credential/README.md

```diff
@@ -3,57 +3,79 @@
+
+### Messages
+
 The Issue Credential protocol consists of these messages:
 
-* Propose Credential - Prover to Issuer (optional)
-* Offer Credential - Issuer to Prover (optional for some credential implementations; required for Hyperledger Indy)
-* Request Credential - Prover to Issuer
-* Issue Credential - Issuer to Prover
+* `propose-credential` - potential Holder to Issuer (optional). Tells what the Holder hopes to receive.
+* `offer-credential` - Issuer to potential Holder (optional for some credential implementations; required for Hyperledger Indy). Tells what the Issuer intends to issue, and possibly, the price the Issuer expects to be paid.
+* `request-credential` - potential Holder to Issuer. If neither of the previous message types is used, this is the message that begins the protocol.
+* `issue-credential` - Issuer to new Holder. Attachment payload contains the actual credential.
 
 In addition, the [`ack`](../0015-acks/README.md) and [`problem-report`](../0035-report-problem/README.md) messages are adopted into the protocol for confirmation and error handling.
 
-This protocol is about the messages to support issuing verifiable credentials, not about the specifics of particular verifiable credential mechanisms. This is challenging since at the time of writing this version of the protocol, there is only one supported verifiable credential mechanism (Hyperledger Indy). [DIDComm attachments](../../concepts/0017-attachments/README.md) are deliberately used in messages to try to make this protocol agnostic to the specific verifiable credential mechanism payloads. Links are provided in the message data element descriptions to details of specific verifiable credential implementation data structures.
+>Note: This protocol is about the messages that must be exchanged to issue verifiable credentials, NOT about the specifics of particular verifiable credential schemes. [DIDComm attachments](../../concepts/0017-attachments/README.md) are deliberately used in messages to isolate the protocol flow/semantics from the credential artifacts themselves as separate constructs. Attachments allow credential formats and this protocol to evolve through versioning milestones independently instead of in lockstep. Links are provided in the message descriptions below, to describe how the protocol adapts to specific verifiable credential implementations.
 
-Diagrams in this protocol were made in draw.io. To make changes:
+#### Choreography Diagram
+
+<blockquote>
+Note: This diagram was made in draw.io. To make changes:
```

Comment:
Thanks for adding these details in here so it's easier for people to reproduce the diagrams and make edits.